SingularityUI/app/templates/taskDetail/taskMetadataTable.hbs

```diff
@@ -0,0 +1,76 @@
+{{#if data.taskMetadata}}
+    <div class="page-header">
+        <h2>
+            Metadata
+            <small>
+                <a data-action="expandToggle">
+                    {{#if expanded}}
+                        collapse
+                    {{else}}
+                        view
+                    {{/if}}
+                </a>
+            </small>
+        </h2>
+    </div>
+    <div class='{{#unless expanded}}hide{{/unless}}'>
+        <div class="table-container">
+            <table class="table table-striped">
```

Comment:
We should check to see what some of the longer/more detailed metadata items look like in a table. Since we had to reformat the error/warn ones to look nicer in the panels at the top, might be worth the same investigation for the info ones